hibernate-core/src/main/java/org/hibernate/sql/results/graph/entity/AbstractEntityInitializer.java

```diff
@@ -530,7 +531,20 @@ else if ( entityInstanceFromExecutionContext != null ) {
 		}
 	}
 
+	protected void registerLoadingEntityInstanceFromExecutionContext(
+			RowProcessingState rowProcessingState,
+			Object instance) {
+	}
```

Comment:
Can you please add a comment here that explains why we don't want to register the entity instance?